server/container_create.go

```diff
@@ -351,6 +353,30 @@ func setupCapabilities(specgen *generate.Generator, caps *types.Capability, defa
 		}
 	}
 
+	if addDefaultCapbilities {
```

Comment:
```suggestion
	if !addAll && !dropAll {
```

to make `addDefaultCapbilities` obsolete